### PR TITLE
Added tabs to the action pane

### PR DESCRIPTION
**Pull request by AlexDorian7**
Added tabs to the action pane to organize all action packs better.

image

### CODE REVIEW

**Comment by rvion** [comment]
thanks again for the contribution !

Just changed a bunch of things there a few minutes ago x)
I'll test tomorrow, about to go to sleep right now
I'm not 100% sure what I want for this panel, but I agree having a way to gropu things can be practical.

by the way, did you notice that once you setup your github username in the navbar, it moves your action packs at the top ?

File: index.html [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
-        <title>🛋️ CushyStudio</title>
-    </head>
-    <body>
-        <div id="root"></div>
-        <script type="module" src="/src/front/ui/main.tsx"></script>
-    </body>
-</html>
+
+<head>
+    <script src="http://localhost:8097"></script>
```
Comment (rvion) [inline index.html:5]: what is this added thing ?

Comment (AlexDorian7) [reply index.html:5]: Ah that, that is used for the react dev tools. When they are not running nothing should happen.

**Comment by rvion** [comment]
@AlexDorian7 didn't take the time to properly address this PR yet, as I was focusing on documentation and core renaming to go toward more stability. just in case you're affected, I renamed the root `actions` folder to `library` as well as rewrote the readme. There is a new big `_prefab` example that shows how to modularise actions here `library/CushyStudio/default/_prefabs.ts` (possibly a few more naming changes, but things are getting way way better)

**Comment by rvion** [comment]
you can also take a look at the new readme, it should be a bit better / a step in the right direction

**Comment by rvion** [comment]
@AlexDorian7 I'll do something along those lines very soon ! By the way, wanna join the discord so we can speak a bit so I can better prioritize features you need ?

**Pull request closed without merging**